js/main.js

```diff
@@ -1,19 +1,75 @@
+const SIMILAR_AD_COUNT = 10;
+const TYPE_OF_HOUSING = ['palace', 'flat', 'house', 'bungalow', 'hotel'];
+const CHECKIN_TIME = ['12:00', '13:00', '14:00'];
+const CHECKOUT_TIME = ['12:00', '13:00', '14:00'];
+const FEATURES = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
+const PHOTOS = ['https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg', 'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg', 'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg'];
```

Comment:
плохо такое писать в одну строку, да как и предыдущие массивы тоже. Лучше вот так

```suggestion
const PHOTOS = [
  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg'
];
```

Reply:
Понял, исправил.